tox.ini

```diff
@@ -44,14 +43,51 @@ deps=
     eth-hash[pycryptodome]
     typing1: eth-typing<2
 
-[testenv:lint]
-basepython=python3
+[common-lint]
+basepython=python
 extras=lint
+allowlist_externals=black
 commands=
```

Comment:
`isort` is a linting dep, but isn't actually called as part of the lint process. I'll be adding that as a separate pr.

Reply:
nm, ran `isort` and it only hit 5 files. fixed and added to this pr.